src/Accelerators/NNPA/CMakeLists.txt

```diff
@@ -19,6 +19,9 @@ else()
   set(ZDNN_LIBRARY_ENABLED 0)
 endif()
 
+include(zdnn.cmake)
+setup_zdnn(v0.4.0)
```

Comment:
We have moved to v1.0.0

Reply:
OK. I will move to v1.0.0 and see if I can fix the zdnn.h problem.